package.json

```
@@ -1,20 +1,21 @@
 {
-  "name": "css-frameworks",
-  "version": "1.0.0",
+  "name": "css-framework-v.2",
+  "version": "2.0.0",
```

```suggestion
  "version": "1.0.1",
```
Version numbers go Major.Minor.Patch where major version represents a substantial change